resources/qml/UserProfile.qml

```diff
@@ -34,279 +34,287 @@ ApplicationWindow {
         onActivated: userProfileDialog.close()
     }
 
-    ColumnLayout {
-        id: contentL
 
-        anchors.fill: parent
-        anchors.margins: 10
-        spacing: 10
-
-        Avatar {
-            url: profile.avatarUrl.replace("mxc://", "image://MxcImage/")
-            height: 130
-            width: 130
-            displayName: profile.displayName
-            id: displayAvatar
-            userid: profile.userid
-            Layout.alignment: Qt.AlignHCenter
-            onClicked: TimelineManager.openImageOverlay(profile.avatarUrl, "")
-
-            ImageButton {
-                hoverEnabled: true
-                ToolTip.visible: hovered
-                ToolTip.text: profile.isGlobalUserProfile ? qsTr("Change avatar globally.") : qsTr("Change avatar. Will only apply to this room.")
-                anchors.left: displayAvatar.left
-                anchors.top: displayAvatar.top
-                anchors.leftMargin: Nheko.paddingMedium
-                anchors.topMargin: Nheko.paddingMedium
-                visible: profile.isSelf
-                image: ":/icons/icons/ui/edit.png"
-                onClicked: profile.changeAvatar()
+    ListView {
```

Comment:
Can you add the scrollhelper to this? It is reeeeaaally hard to use on my touchpad :D